docs/api/cases/cases-api-get-reporters.asciidoc

```diff
@@ -0,0 +1,60 @@
+[[cases-api-get-reporters]]
+== Get reporters API
+++++
+<titleabbrev>Get reporters</titleabbrev>
+++++
+
+Returns all case reporters (users who opened cases).
+
+=== Request
+
+`GET <kibana host>:<port>/api/cases/reporters`
+
+`GET <kibana host>:<port>/s/api/cases/reporters`
+
+=== Prerequisite
+
+You must have `read` privileges for the *Cases* feature in the *Management*,
+*{observability}*, or *Security* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>, depending on the
+`owner` of the cases you're seeking.
+
+=== Query parameters
+
+`owner`::
+(Optional, string) A filter to limit the retrieved reporters to a specific set 
```

Comment:
It is `Optional, string or array of strings`. Check here: https://kibana_129089.docs-preview.app.elstc.co/guide/en/kibana/master/cases-api-get-status.html. I noticed that I missed that in https://kibana_129089.docs-preview.app.elstc.co/guide/en/kibana/master/cases-api-get-tag.html which also should be `Optional, string or array of strings`. Maybe we can fix it in this PR if you like. Usually, the type of the owner is `string or an array of strings` when you do find queries `string` only when you create something.

Reply:
@cnasikas Thanks, I updated the PR accordingly via [28c235e](https://github.com/elastic/kibana/pull/129089/commits/28c235e067c2198bf784ee4cdba23d0fa484cf31).